Instrukcije_d001/src/domain/Zeton.h

```diff
@@ -0,0 +1,17 @@
+//
+// Created by Edin on 15. 03. 2022.
+//
+
+#ifndef INSTRUKCIJE_D001_ZETON_H
+#define INSTRUKCIJE_D001_ZETON_H
+
+typedef struct Zeton {
+    int x;
+    int y;
+    char XorO;
```

Comment:
Tukaj bi jaz raje dal XorO bolj splošno ime... `znak`. Ker to zgleda kot da se shranjuje boolean.